db/db_impl.cc

```diff
@@ -1029,6 +1029,13 @@ Status DBImpl::GetImpl(const ReadOptions& read_options,
   auto cfh = reinterpret_cast<ColumnFamilyHandleImpl*>(column_family);
   auto cfd = cfh->cfd();
 
+  if (tracer_) {
+    InstrumentedMutexLock lock(&trace_mutex_);
+    if (tracer_) {
+      tracer_->Get(column_family, key);
```

Comment:
It's OK for now but maybe add a TODO to remove this mutex in the future. This can be a significant performance penalty which prevents people from enabling it.